### PR TITLE
ES-1974

### PR DESCRIPTION
**Pull request by wrighbr**

### CODE REVIEW

**Review by ChristianGroentved — APPROVED** [review]
LGTM

File: template/.circleci/config.yml [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
+      - cci-common/build_n_push_docker:
+          context: es02-dev
+          filters:
+            branches:
+              only: master
+          requires:
+            - secret-injector/dump-secrets
+      - cci-common/deploy_to_gke:
+          context: es02-dev
+          files_to_deploy: ./src/tests/k8s/deployment.yml
```
Comment (brondum) [inline template/.circleci/config.yml:27]: This wont work.
that should either be ./deployment.yml or just deployment.yml

Comment (wrighbr) [reply template/.circleci/config.yml:27]: lol I just copied n pasted and didn't look all fixed now

File: template/.circleci/config.yml [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^\d+\.\d+\.\d+$/
+          requires:
+            - secret-injector/dump-secrets
+      - cci-common/deploy_to_gke:
+          context: es02-dev
+          files_to_deploy: ./src/tests/k8s/deployment.yml
+          files_to_replace_strings: ./src/tests/k8s/deployment.yml
```
[the remark this answers is not in the document]
Comment (wrighbr) [reply template/.circleci/config.yml]: lol I just copied n pasted and didn't look all fixed now

File: template.yaml [hunk 3]
```diff
@@ -74,9 +75,27 @@ spec:
...
+      if: ${{ parameters.deployment_type !== "api" }}
+      input:
+        files:
+          - api/
+          - docs/docs.go
+          - docs/swagger.yaml
+    - id: remove-techdocs-files
+      name: remove files
+      action: fs:delete
+      if: ${{ parameters.use_techdocs !== true }}
```
[the remark this answers is not in the document]
Comment (wrighbr) [reply template.yaml]: nice caught all fixed up

File: template/.circleci/config.yml (hunk 1 above)
Comment (Gaardsholt) [inline template/.circleci/config.yml:27]: isn't this path wrong?

File: template/.circleci/config.yml [hunk 4]
```diff
@@ -1,42 +1,62 @@
...
+          context: es02-dev
+          filters:
+            branches:
+              only: master
+          requires:
+            - secret-injector/dump-secrets
+      - cci-common/deploy_to_gke:
+          context: es02-dev
+          files_to_deploy: ./src/tests/k8s/deployment.yml
+          files_to_replace_strings: ./src/tests/k8s/deployment.yml
```
Comment (Gaardsholt) [inline template/.circleci/config.yml:28]: isn't this path wrong?

File: template/.circleci/config.yml (hunk 1 above)
Comment (wrighbr) [reply template/.circleci/config.yml]: fixed now

File: template/.circleci/config.yml (hunk 4 above)
Comment (wrighbr) [reply template/.circleci/config.yml]: fixed now

**Pull request merged**